packages/core/test/specs/editor/telemetry.ts

```diff
@@ -0,0 +1,126 @@
+import Editor from '../../../src/editor';
+
+describe('Editor telemetry', () => {
+  let editor = new Editor();
+  let originalFetch: typeof fetch;
+  let fetchMock: jest.Mock;
+
+  beforeEach(() => {
+    originalFetch = global.fetch;
+    fetchMock = jest.fn(() => Promise.resolve({ ok: true }));
+    global.fetch = fetchMock;
+
+    const sessionStorageMock = {
+      getItem: jest.fn(),
+      setItem: jest.fn(),
+      removeItem: jest.fn(),
+    };
+
+    Object.defineProperty(window, 'sessionStorage', { value: sessionStorageMock });
+
+    console.log = jest.fn();
+    console.error = jest.fn();
+  });
+
+  afterEach(() => {
+    global.fetch = originalFetch;
+    jest.resetAllMocks();
+  });
+
+  const triggerLoadAndWait = (editor: Editor) => {
+    return new Promise<void>((resolve) => {
+      editor.on('telemetry:sent', () => {
+        resolve();
+      });
+      editor.getModel().trigger('load');
+    });
+  };
+
+  test('Telemetry is sent when enabled', async () => {
```

Comment:
Only this test in the file is working, the rest hang around - something to do with timers, hoping to shed light on what events to listen to from this comment: 

* https://github.com/GrapesJS/grapesjs/pull/6210/files#r1796327194

Reply:
Check this one https://github.com/GrapesJS/grapesjs/pull/6195

Reply:
Thanks, moving to editor view load and using your test utils works.